### PR TITLE
fix(general-petitions): pagination fix service portal, and a few minor changes in web

### PR DESCRIPTION
**Pull request by albinagu**
## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by jGeir — APPROVED** [review]
LGTM

File: apps/web/screens/PetitionView/PetitionView.tsx [hunk 1]
```diff
@@ -66,98 +68,121 @@ const PetitionView = () => {
...
-              </Box>
+                      {n(
+                        'putMyNameOnThatList',
+                        'Setja nafn mitt á þennan lista',
+                      )}
+                    </Button>
+                  </GridColumn>
+                </GridRow>
+                <GridRow marginTop={5} marginBottom={5}>
+                  <GridColumn span={['12/12', '12/12', '12/12']}>
```
Comment (alexandradilja) [inline apps/web/screens/PetitionView/PetitionView.tsx:129]: `<GridColumn span={'12/12'}>`

**Review by alexandradilja — APPROVED** [review]
one tiny comment, otherwise lgtm

File: apps/web/screens/PetitionView/PetitionView.tsx (hunk 1 above)
Comment (albinagu) [reply apps/web/screens/PetitionView/PetitionView.tsx:129]: já lol

**Pull request merged**